src/composables/useColourScale.ts

```diff
@@ -53,7 +53,12 @@ export const useColourScale = (selectedIndicators: Ref<Dict<FeatureIndicatorValu
         return result;
```

Comment:
in indicatorExtremes, line 51, when we define the default range:
```ts
indicatorRange.range =
                indicatorRange.max !== indicatorRange.min // Avoid dividing by zero if only one value...
                    ? indicatorRange.max - indicatorRange.min
                    : 1;
```

Does it matter that a range of 1 is actually quite a large range when we're working with certain metrics? A typical range for FOI is on the order of 0.02 and can be smaller.

Reply:
We should possibly include a 'default range' for indicators. But I think in practice this issue is very unlikely to come up since we'll basically never have only a single region to display. We inherited this bit of logic from HINT where it can happen, but it's possibly overkill here.

Reply:
Maybe
```ts
indicatorRange.range = indicatorRange.max - indicatorRange.min
```
is better then, for concision. Doesn't need to happen in this branch though.